helpers/DeliveryHelper.php

```diff
@@ -557,9 +559,12 @@ public static function setHasBeenPaused($deliveryExecution, $paused)
         }
         /** @var DeliveryMonitoringService $deliveryMonitoringService */
         $deliveryMonitoringService = ServiceManager::getServiceManager()->get(DeliveryMonitoringService::SERVICE_ID);
-        $data = $deliveryMonitoringService->getData($deliveryExecution);
+
+        $data = new DeliveryMonitoringData($deliveryExecution, []);
```

Comment:
`new DeliveryMonitoringData` statement used 17 times. I would like to not use `new` statement to instantiate this object, but use factory method as it was previously.

Reply:
done